specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/examples/ApiManagementGetKubernetesManagedWorkspace.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2024-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "workspaceId": "wks1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/workspaces/wks1",
+        "type": "Microsoft.ApiManagement/service/workspaces",
+        "name": "wks1",
+        "kind": "KubernetesManaged",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "description": "workspace 1",
+          "displayName": "my workspace",
+          "configurationApi": {
+            "hostname": "example.eastus.configuration.gateway.azure-api.net"
```

Comment:
New format should not have gateway and format is similar to `<per-tenant>.configuration.<region>-01.azure-api.net`

Let's make sure we update all samples